pkg/policy/policies.go

```diff
@@ -296,14 +296,12 @@ func (ps *Policies) Clone() utils.Cloner {
 		if p == nil {
 			continue
 		}
-		if err := nPols.Add(p.Clone().(*Policy)); err != nil {
+		if err := nPols.Set(p.Clone().(*Policy)); err != nil {
```

Comment:
With the new generic change the type inference can be dropped right?

Reply:
Yep, must be. I'll rebase it. Tks.